scorep/__main__.py

```diff
@@ -11,6 +12,39 @@ def _err_exit(msg):
     sys.exit(1)
 
 
+def print_help():
+    print("""
+Usage: python -m scorep [options] [--] your_program.py [args]
+
+Score-P Python instrumentation wrapper. The following options control how the program is instrumented and executed:
+
+  --help                   Show this help message and exit.
+  --mpi                    Enable MPI instrumentation (equivalent to --mpp=mpi).
+  --keep-files             Keep temporary files after execution.
+  --verbose, -v            Enable verbose output for debugging and tracing.
+  --nocompiler             Disable compiler-based instrumentation.
+  --nopython               Disable instrumentation of Python code.
+                           Instrumentation can still be enabled later from within the application's source code.
+  --noinstrumenter         Same as --nopython.
```

Comment:
time to deprecate that option?

Reply:
add `--instrumenter-type=none` in the future? and make `--noinstrumenter` an alias?